.github/workflows/reusable_monitoring.yml

```
@@ -65,7 +65,7 @@ jobs:
           ref: "${{ needs.detect-workflow.outputs.ref }}"
       - uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4.0.1
         with:
-          go-version: 1.19
+          go-version: 1.20
```

opsy

this needs to be in quote

```suggestion
          go-version: '1.20'
```

otherwise will get the go 1.2